addons/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/handler/AbstractModbusEndpointThingHandler.java

```diff
@@ -37,6 +37,16 @@
 public abstract class AbstractModbusEndpointThingHandler<E extends ModbusSlaveEndpoint, C> extends BaseBridgeHandler
         implements ModbusManagerListener, ModbusEndpointThingHandler {
 
+    protected static class ModbusConfigurationException extends Exception {
```

Comment:
Could you please extract this into a separate class file?

Reply:
Done